devsetup/tripleo/tripleo_install.sh

```diff
@@ -94,6 +109,6 @@ openstack overcloud deploy --stack overcloud \
     -e /home/zuul/containers-prepare-parameters.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/podman.yaml \
     -e /usr/share/openstack-tripleo-heat-templates/environments/low-memory-usage.yaml \
     -e /usr/share/openstack-tripleo-heat-templates/environments/debug.yaml --validation-warnings-fatal ${ENV_ARGS} ${CEPH_OVERCLOUD_ARGS} \
-    -e /home/zuul/overcloud_services.yaml -e /home/zuul/config-download.yaml \
+    -e /home/zuul/overcloud_services.yaml -e /home/zuul/$config_download \
```

Comment:
if I understand correctly, now we have a `ComputeCount: 3` in `overcloud_services` and a `ComputeCount: 2` in the new `config-download-networker`. Do we know how the precendence is handled? Is it like ansible when only the last occurrence counts?

Reply:
yes exactly
with the tripleo-heat-templates environments last one wins and I am relying on that config-download-networker.yaml to come after services.yaml

it allows us to keep it simpler here with the networker vars in one file

Reply:
ack, in that case maybe I would left a comment noting that we rely on that order to get the right amount of computes, for future reference, other than that, the PR lgtm